packages/js/src/ui/components/primitives/Tooltip/TooltipContent.tsx

```diff
@@ -0,0 +1,51 @@
+import { JSX, onCleanup, onMount, Show, splitProps } from 'solid-js';
+import { Portal } from 'solid-js/web';
+import { AppearanceKey, useFocusManager } from '../../../context';
+import { useStyle } from '../../../helpers';
+import { Root } from '../../elements';
+import { useTooltip } from './TooltipRoot';
+
+export const tooltipContentVariants = () =>
+  'nt-bg-neutral-alpha-900 nt-p-4 nt-shadow-[0px_5px_20px_0px_rgba(0,0,0,0.20)] nt-z-10 nt-rounded-lg nt-text-background nt-text-sm';
+
+const TooltipContentBody = (props: TooltipContentProps) => {
+  const { open, setFloating, floating, floatingStyles } = useTooltip();
+  const { setActive, removeActive } = useFocusManager();
+  const [local, rest] = splitProps(props, ['class', 'appearanceKey', 'style']);
+  const style = useStyle();
+
+  onMount(() => {
+    const floatingEl = floating();
+    setActive(floatingEl!);
+
+    onCleanup(() => {
+      removeActive(floatingEl!);
+    });
+  });
+
+  return (
+    <div
+      ref={setFloating}
+      //id is necessary here because this is a portal
```

Comment:
❓ **question:** is something missing?

Reply:
Ah, no. It's a leftover comment in the popover as well.